backend/seed.ts

```diff
@@ -160,6 +231,56 @@ async function main() {
     },
   })
 
+  // Add comment to Jon's post
+  await db.post.update({
+    where: {
+      id: 2,
```

Comment:
Using a hardcoded ID here probably isn't ideal, it could change based on when the seed data is loaded, or the order in which things are added in the seed data. The fastest way to add add threads is in the same prisma call that creates the post (same way a thread is created for the other one), but if you do want to make this thread separately you'll want to save the post ID above when it's created and reference that.

Reply:
Yeah I couldn't do it in the same place as creating the post because that would mean that the other users weren't created yet and so it gave errors for referencing vars before they were declared

Reply:
There we go. I changed the structure a little so that we start by creating the users, and then create the post afterwards along with comment threads and that way the users are all created and available to interact! 😄